Auth0/SafariAuthenticationSession.swift

```diff
@@ -64,11 +67,11 @@ class SafariAuthenticationSession: AuthSession {
 
     private var authSession: AuthenticationSession?
 
-    init(authorizeURL: URL, redirectURL: URL, state: String? = nil, handler: OAuth2Grant, finish: @escaping FinishSession, logger: Logger?) {
+    init(authorizeURL: URL, redirectURL: URL, state: String? = nil, handler: OAuth2Grant, finish: @escaping FinishSession, logger: Logger?, presentationContextController: Any? = nil) {
```

Comment:
Where is the presentationContextController parameter used?  I see a webAuthenticationSession.presentationContextProvider set statement on line 88 but it sets to self.

Reply:
Good catch, this was left over from refactoring this code. It's no longer required as this class now implements the protocol via an extension at https://github.com/auth0/Auth0.swift/blob/added-ios13-webauth-compatibility/Auth0/SafariAuthenticationSession.swift#L141